Update GitHub Action documentation (#5)
sclorg/testing-farm-as-github-action: README.md

# testing-farm-as-github-action

> Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.

Reviewer: ```suggestion
Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
```
Testing Farm as a GitHub Action is a GitHub Action for executing tests against specific Pull Requests on the Testing Farm infrastructure. The GitHub action reports the result of the tests with Pull Request status.

> This action only runs on `Linux`. Before a calling this GitHub Action, you must first clone your project by e.g. 

Reviewer: ```suggestion
This action internally runs on ubuntu-20.04. Before calling this GitHub Action, you must first clone your project by, e.g., 
```
I have checked the code, and it does not seem, that the user can choose the base Linux distribution for this action. I would say, that is sufficient to state, that the action runs on ubuntu.

Reviewer: Interesting, good to know, thanks :).

Author: Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/6

> by GitHub Action [Checkout V2](https://github.com/actions/checkout).

Reviewer: ```suggestion
GitHub Action [Checkout V2](https://github.com/actions/checkout).
```

The Action uses ubuntu-20.04.

Important binaries, like curl, jq, which are used for getting scheduled jobs from Testing Farm are already installed
by this GitHub Action.
 
In your organization secrets you should have stored API key to testing farm.

## Action Inputs

| Input Name      | Description             | Default |
|-----------------|-------------------------| ------- |
| api_key         | A testing farm API key. | **Must be provided** |  
| tmt_repository | An url to tmt repository | **Must be provided** |

> | test_fmf_plan | A fmf plan which will be selected. | All plans |

Reviewer: What means "All plans"? It is not clear from this table.

Author: It means all plans in fmf plan.
